src/surface.cpp

```diff
@@ -1156,9 +1196,10 @@ void read_surfaces(pugi::xml_node node)
   }
 
   // Loop over XML surface elements and populate the array.  Keep track of
-  // periodic surfaces.
+  // periodic surfaces and their albedos. A pair of physically meaningful
+  // periodic surfaces can only have one albedo value.
   model::surfaces.reserve(n_surfaces);
-  std::set<std::pair<int, int>> periodic_pairs;
+  std::set<std::pair<std::pair<int, int>, std::double_t>> periodic_pairs;
```

Comment:
I did this to ensure I don't mess up the albedos between pairs of periodic surfaces because there is rarely only one pair. How would you suggest I rewrite this?

Reply:
I would suggest adding an `std::map` that maps surface IDs to albedo values. Then when the PeriodicBC objects are created, try to lookup albedo values from the map. Does that make sense?